lib/services/azurerediscache/index.js

```diff
@@ -30,8 +30,8 @@ Handlers.provision = function(log, params, next) {
   if (!cp.allValidatorsSucceed()) {
     var reply = Reply(HttpStatus.INTERNAL_SERVER_ERROR);
     reply.value.description = 'Input validators failed.';
-    var err = new Error('Validation failed.');
-    err.code = HttpStatus.INTERNAL_SERVER_ERROR;
+    var err = new Error('Parameter validation failed.  Did you supply a redis cache parameters file?');
+    err.statusCode = HttpStatus.INTERNAL_SERVER_ERROR;    
```

Comment:
Fixed.  But can you explain to me why this is important?  I also fixed a few more err.code instances in the same file.